setup.py

```diff
@@ -59,7 +59,7 @@ def run_tests(self):
     packages=find_packages(),
     include_package_data=True,
     setup_requires=['wheel'],
-    install_requires=['sphinx>=1.1.3,<2',
```

Comment:
if possible I wouldn't drop support to exsiting versions, unless that's completely necessary.

Reply:
I have no idea what is possible and what is not. The old requirement of >=1.1.3 and <2 already spans versions where sphinx seemingly switched from method of logging to another (between 1.5 and 1.6 there appears to be a change in logging).

According to this https://github.com/sphinx-doc/sphinx/issues/4375#issuecomment-355316362 (and likely other places as well) Sphinx 2.0 and forward will be Python3 only. So likely 1.7 and 1.8 were for both python2 and python3. 

Maybe it is only worth supporting back to releases supporting python3? People can still install an old version if they really need this extension with sphinx 1.1 ...